hermes/src/network/p2p.go

```diff
@@ -46,122 +46,123 @@ import (
 
 //export RegisterObservationCallback
 func RegisterObservationCallback(f C.callback_t) {
-    go func() {
```

Comment:
seems like this is mostly formatting? can we add this to pre-commit to keep the diffs clean?

Reply:
Well. it seems that go strongly suggests formatting using tabs and it is enforced by gofmt (and it's not easily customisable)